### PR TITLE
Fixed wording for mentee application field

### PR DESCRIPTION
**Pull request by BryceSlade**
## Description

This is a very small PR to fix a bug within the Mentee application component. The label of the last field should be; "Anything else you want us to know?". Instead, it was: "Anything else you want us to know? community?". I've removed "community?". This was more than likely altered in a previous PR. 

#### Video Link

[Loom Video](https://www.loom.com/share/1e7a60247f2b48e28d6228dff7ff412c)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have removed unnecessary comments/console logs from my code
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] No duplicate code left within changed files
- [X] Size of pull request kept to a minimum
- [X] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by prashishb — APPROVED** [review]
Simple PR and clean code. Good catch!

File: src/components/pages/Profile/EditProfile.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
-import React, { useState, useEffect } from 'react';
```
Comment (morgankj) [inline src/components/pages/Profile/EditProfile.js:1]: THANK YOU for removing this, these errors kill me

File: src/components/pages/RoleSignup/Applications/Mentee.js [hunk 2]
```diff
@@ -445,7 +445,7 @@ const Mentee = () => {
 
                 <Col md={22} xs={24}>
                   <Form.Item
-                    label="Anything else you want us to know? community?"
+                    label="Anything else you want us to know?"
```
Comment (morgankj) [inline src/components/pages/RoleSignup/Applications/Mentee.js:448]: Simple and clean fix - excellent catch

**Review by morgankj — APPROVED** [review]
Nice easy changes! Great catch to keep code dry, clean, and minimize errors

**Pull request merged**